tests/jobs.py

```diff
@@ -1794,6 +1796,8 @@ def test_execute_task_instances_limit(self):
             ti.refresh_from_db()
             self.assertEqual(State.QUEUED, ti.state)
 
+    @unittest.skipUnless("INTEGRATION" in os.environ,
```

Comment:
So that's effectively switching off these tests as far as travis goes right? What effect does it have on coverage?